.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vcpkg"]
+	path = vcpkg
+	url = https://deliveryoptimization.visualstudio.com/client/_git/vcpkg
```

Comment:
Hmm, this is a private repo. I think we don't need this anymore right? The windows build does not depend on any other 3rd party libs.

Reply:
Ah I had deleted the submoduled directory but not the .gitmodules file

Yes it can be removed, but note that we sitll use dependencies for our unit tests (gtest, and i think maybe some boost::filesystem functions)

The reason I had added vcpkg as a submodule originally was to make building on a msft-hosted agent easier. But we now have the vcpkg caching solution.

So will remove